### DISCUSSION TITLE
BallotMaker documentation

### DISCUSSION THREAD

**Discussion by stratofax** [TrustTheVote-Project/BallotLab#120] [Ideas]
Based on the excellent documentation for the [TrustTheVote-Project/MarkIt-Client-Mobile-App](https://github.com/TrustTheVote-Project/MarkIt-Client-Mobile-App) repo, it's time to up our documentation game here at the BallotLab. Some ideas:

Solid documentation for everything related to ballots. Examples of things we think the client might want written down in some fashion:

- explanation of field types / IDs
- expectation of the client when marking a contest selection (checkbox)
- expectation of the client when marking a write-in
- etc.

Include in the project's READ.ME file a link to [Cliff's EDF documentation about which subset of the EDF schema is implemented in this project](https://github.com/TrustTheVote-Project/NIST-1500-100-103-examples/blob/main/profile/profile.org)

**Comment by awoflaherty** [comment]
In your documentation, please link to this:
https://github.com/TrustTheVote-Project/MarkIt-Client-Mobile-App/blob/main/libs/services/pdf/README.md#affidavit-pdfs
and advise Matt or me if anything in the link is not accurate.

**Reply by stratofax** [reply]
@awoflaherty I don't know much about affidavit PDFs. Better to link here, perhaps? https://github.com/TrustTheVote-Project/MarkIt-Client-Mobile-App/blob/main/libs/services/pdf/README.md#ballot-pdfs
